### ISSUE TITLE
[Question] Padding transforms can only deal with 2d data?

### ISSUE THREAD

**Issue by neuronflow** [Project-MONAI/MONAI#995]
If I get it right the padding transforms can only deal with 2D data? A 3d version would be highly appreciated.
https://docs.monai.io/en/latest/transforms.html#spatialpadd
https://docs.monai.io/en/latest/transforms.html#divisiblepadd
https://docs.monai.io/en/latest/transforms.html#borderpadd

At least when I apply them they only affect one or two dimensions depending on the parameters I provide.

**Comment by Nic-Ma** [comment]
Hi @neuronflow ,

Thanks for your interest here.
All the 3 padding transforms can support both 3D and 2D data.
You can find how to set parameters in the doc-strings of these classes.
For example: set the `spatial_size` in `SpatialPadd`, `spatial_border` in `BorderPadd `, `k` in `DivisiblePadd`.

Thanks.

**Comment by neuronflow** [comment]
Thanks @Nic-Ma !

Hmm I spent like 30minutes trying to get my `[1,4, 155, 240, 240]` sized tensor to  size `[1, 4, 256 ,256 , 256]` using the Pad Transforms mentioned above.

The best I could achieve was a [1, 4, 155, 256, 256] image. Unfortunately I don't have all the code variants I tried anymore.

I now wrote my own transform to solve it but I would still be interested to find out which parameters I should use in MONAI to achieve the same :)
